lua/lvim/core/bufferline.lua

```diff
@@ -152,6 +152,8 @@ M.setup = function()
     return
   end
 
+  vim.opt.showtabline = 2
+
```

Comment:
why not move is settings,lua?

Reply:
when it was in settings, there was flickering when default tabline changed to bufferline when it's loaded